src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractListItemFactory.cs

```diff
@@ -661,8 +661,9 @@ public ImmutableArray<ObjectListItem> GetReferenceListItems(ObjectListItem paren
                 return ImmutableArray<ObjectListItem>.Empty;
             }
 
-            var builder = ImmutableArray.CreateBuilder<ObjectListItem>();
+            var builder = ArrayBuilder<ObjectListItem>.GetInstance();
 
+            // Don't show previous script compilations in the references list.
```

Comment:
@jasonmalinowski , you're just stating an even stronger reason why no change is required here, correct?